Makefile

```diff
@@ -171,7 +177,11 @@ run_tests.acceptance: ## Runs the acceptance tests for the parsed suite
 	@echo '------------------------------------------------------'
 	@echo '  Starting acceptance test for the suite: $(suite)'
 	@echo '------------------------------------------------------'
-	./node_modules/.bin/jasmine --config="./core/tests/puppeteer-acceptance-tests/jasmine.json" ./core/tests/puppeteer-acceptance-tests/spec/$(suite) || $(MAKE) stop
+	@if [ -d ./core/tests/puppeteer-acceptance-tests/build ]; then \
```

Comment:
Why do we need the backslashes at the end here, but not at the end of previous lines? Or does shell not recognize an if/else unless you cram everything into the same "line"?

Reply:
It's `Makefile`, Makefile syntax requires commands to be in a single line. So, to use multiple lines, for readability, we need to use `\`